### PR TITLE
Lich King Vision

### PR DESCRIPTION
**Pull request by zumbak04**
If you're able to review English localization, please follow [This Link](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/pull/1053/files?file-filters%5B%5D=.csv#diff-177f28deabd40e49753f6ba47a622790) to see localization files and check [This Guide](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/wiki/How-to-be-Keeper-(Tester)#reviewing-code) to see how it can be reviewed.

## Changelog:
- Added an event chain where you travel to the heart of Northrend at the call of the mystical Lich King, embracing **Death God** as your religion.
- Non-evil stubborn people are likely not to do evil doubtful actions like drinking demon blood, summoning demons etc.

## Scripting changelog (ignore if you can't read CK2 script):
- Purged unused `fel_presence` in `WCFEL.9900`.

# How to test:
Start in Northrend of the 603 bookmark, use `event WCUND.9900` console command and watch what happens. If the event didn't fire instantly, try to fire it again.

### CODE REVIEW

File: localisation/00_zuwc_united_localisation.csv [hunk 1]
```diff
@@ -2876,13 +2876,13 @@ EVTTITLE_WCUND_1010;End of the Scourge;;;;;;;;;;;;;x
...
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
+EVTDESC_WCUND_9910;In your dreams you see the throne hidden in the heart of Northrend. The creature that sits on it calls you, promising to deliver you from the agony of mortal life.;;;;;;;;;;;;;x
+EVTOPTA_WCUND_9910;Alright. We're going to find the creature.;;;;;;;;;;;;;x
+EVTOPTB_WCUND_9910;Seems too dangerous.;;;;;;;;;;;;;x
+EVTDESC_WCUND_9915;With each day of your journey to the heart of Northrend, the blizzard was getting stronger. Through the veil of snow, you see black figures surrounding you. Once they were living beings, now nothing more than rotting corpses. The creature on the throne looks at you with their blue glowing eyes. Tired, you fall to your knees as the living dead approach you.;;;;;;;;;;;;;x
```
Comment (zumbak04) [inline localisation/00_zuwc_united_localisation.csv]:
```suggestion
EVTDESC_WCUND_9915;With each day of your journey to the heart of Northrend, the blizzard is getting stronger. Through the veil of snow, you can see black figures surrounding you. Once they were living beings, but now they are nothing more than rotting corpses. The creature on the throne looks at you with its glowing blue eyes. Tired, you fall to your knees as the living dead approach you.;;;;;;;;;;;;;x
```

**Comment by zumbak04** [comment]
> So, event WCUND.9900 did not appear to do anything. Waited for about a year, received no event.

@LordSilvermane forgot to mention that you must be in Northrend it may not fire after one time. Changed the **How to test** section.

> A few changes to localization

Better **Suggest** localization. I tried to explain it [Here](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/wiki/How-to-be-Keeper-(Tester)#reviewing-code). If you have questions, feel free to ask. I suggested your text above to show how it looks. If you want to try, you can suggest your second text.

**Comment by zumbak04** [comment]
@LordSilvermane implemented your texts in https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/pull/1053/commits/a31d3302d2915e737b5e6448ce73f4f5fe319a98.

File: events/wc_undead_events.txt [hunk 2]
```diff
@@ -2737,4 +2737,172 @@ character_event = {
...
+					NOT = { has_character_flag = lich_king_vision_happend_flag }	# Happens once in lifetime
+					
+					lich_king_vision_trigger = yes
+				}
+				set_character_flag = lich_king_vision_happend_flag
+				
+				character_event = { id = WCUND.9910 }	# Lich King vision
+			}
+			1 = {	# Small chance of nothing
+			}
```
Comment (MemeWizard) [inline events/wc_undead_events.txt]: Event is already random, why do you need this superfluous thing?

Comment (zumbak04) [reply events/wc_undead_events.txt]: Purged in https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/pull/1053/commits/bf15f659a7a70b482935703d35ebecedc9367beb.

**Pull request merged**